beta-src/src/webDiplomacyTheme.ts

```diff
@@ -103,6 +113,16 @@ const countryPalette: CountryPaletteOptions = {
   },
 };
 
+const arrowPalette: ArrowPalette = {
+  moveOrderSelected: "#FFFFFF",
```

Comment:
What arrow are you drawing on `moveOrderSelected`? I'm not sure why this is here.

Reply:
This also answers the question above; the task shows selecting to move order, is indeed a move with a white arrow
<img width="716" alt="Screen Shot 2022-02-10 at 1 15 39 PM" src="https://user-images.githubusercontent.com/95884853/156472288-fa2140f2-62e3-4eb0-9ca5-f5af5855dff1.png">

Reply:
@sariacodazen gotcha, maybe the enum `Arrow` or `ArrowType` makes more sense since we also have the move failed arrow.